test/recipes/01-test_symbol_presence.t

```diff
@@ -29,10 +36,21 @@ note
     "ordinals (util/*.num in the source tree), something that should be\n",
     "good enough a check for the other platforms as well.\n";
 
-foreach my $libname (@libnames) {
+my $shlibpath;
+my $libname;
+
+foreach my $thislib (@libnames) {
  SKIP:
     {
-        my $shlibpath = bldtop_file("lib" . $libname . ".so");
+        if ($thislib eq "fips") {
+            skip "FIPS is disabled in this build", 2 if disabled("fips");
+            $shlibpath = bldtop_file("providers", platform->dso('fips'));
+            #Actually we're going to compare it against libcrypto
+            $libname = "crypto";
```

Comment:
Uhmmm, no we are not.  We should compare against `util/providers.num`  Please remember that we're checking *exported* symbols.